### PR TITLE
gluon-web-wifi-config: Fix PHY detection

### PR DESCRIPTION
**Pull request by belzebub40k**
Fix for  #1088. Tested on Linksys WRT1200AC and TP-Link TL-WR841N-v11.

### CODE REVIEW

**Comment by NeoRaider** [comment]
Hmm, I think what LuCI and some other scripts do is to simply try the path once with "platform", and fall back to the path without if that does not work. We should probably do the same.

**Comment by belzebub40k** [comment]
I've updated the PR as requested. This was my initial solution but I thought you prefer code with less duplicates ...

**Comment by NeoRaider** [comment]
Speaking of duplicate code, it would be a good idea to make *find_phy_by_path* and *find_phy_by_macaddr* non-local in gluon.util and remove those functions from the gluon-web-wifi-config model.

**Comment by belzebub40k** [comment]
Updated PR to use `find_phy()` from _gluon-core_.

File: package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua [hunk 1]
```diff
@@ -134,10 +134,14 @@ function get_mesh_devices(uconn)
 	return devices
 end
 
-local function find_phy_by_path(path)
+function find_phy_by_path(path)
```
Comment (NeoRaider) [inline package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua]: Wrong function: you use *find_phy*, but removed the `local` for *find_phy_by_path*.

Comment (belzebub40k) [inline package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua]: Fixed

**Comment by NeoRaider** [comment]
Thanks, I've pushed your changes with some additional refactoring.

**Comment by belzebub40k** [comment]
Thanks. Didn't had time to make the requested changes yesterday.

**Pull request closed without merging**